user_docs/en/userGuide.t2t

```diff
@@ -2502,6 +2502,26 @@ Note that this paragraph style cannot be used in Microsoft Word or Microsoft Out
 
 You may toggle through the available paragraph styles from anywhere by assigning a key in the [Input Gestures dialog #InputGestures].
 
+==== Sentence reconstruction = ====[SentenceReconstruction]
+
+|| . {.hideHeaderRow} | . |
+| Options | Default (Reconstruct across same style paragraphs), Reconstruct across same style paragraphs, Never reconstruct sentences across paragraphs, Reconstruct sentences across any paragraphs|
+| Default | Reconstruct across same style paragraphs |
+
+
+This combo box allows you to select whether NVDA will try to identify and reconstruct sentences spanning across multiple paragraphs when navigating by sentence via``alt+upArrow`` and ``alt+downArrow``.
+Sentence reconstruction comes in handy in two typical use cases:
+- In most Most PDF files text is incorrectly split into paragraphs. Every line typically denotes a paragraph and thus most sentences span many adjacent paragraphs and in order to identify complete sentences NVDA must look into adjacent paragraphs.

```

Comment:
Duplicate word: most (In most Most PDF files...).  Also "Every line typically denotes a...." should be on a new line.  I would also split into two sentences as:

Every line typically denotes a paragraph and thus most sentences span many adjacent paragraphs.
In order to identify complete sentences NVDA must look into adjacent paragraphs.